Uc 1 search1.1: Landing Page Search

lpowell2 opened the pull request:
Please test landing page search. Also verify admin search still works. DON'T MERGE INTO MAIN/APPROVE JUST ADD COMMENTS

jcochran840 committed:
Update personSearch.php

Jeanneveev commented on addTag.php:
```diff
@@ -0,0 +1,283 @@
+
+            <div class="availability-container">
+
+                <div class="availability-day">
```
The radio buttons turn on and off, but not the drop-downs do not activate regardless.

This could be fixed with a PHP if statement to the effect of:
$checked=$_POST['radio_btn_var']
if $checked!=NULL:
     $drop_down_var remove hidden (you may be able to use prop or attr, but I'm not entirely sure)

Jeanneveev commented on addTag.php:
```diff
@@ -0,0 +1,283 @@
+            <p>Please enter the days, times, and frequency of the food bank.</p>
+
+            
+            <label><em>* </em>Availability</label>
```
Availability isn't required, the form can be submitted without it. I'm assuming you purposefully turned it off, though, for the sake of testing

Jeanneveev commented on addTag.php:
```diff
@@ -0,0 +1,283 @@
+        
+
+        <p>By pressing Submit below, the food bank and assciated information you have input will be added to the system.</p>
+        <input type="submit" name="addfb-form" value="Submit">
```
Nothing is added when the form is submitted

Jeanneveev commented on addTagForm.php:
```diff
@@ -0,0 +1,19 @@
+
+<h1>Add a New Tag</h1>
```
The form looks fine, you can submit, but it doesn't change anything in MySQL

Jeanneveev reviewed (commented):
Was made aware that this is not the final branch, will be switching over to the correct one.

jcochran840 committed:
Update dbPersons.php

jcochran840 committed:
Update dbinfo.php

jcochran840 committed:
Merge branch 'dev' of https://github.com/lpowell2/FUMC-FB into dev

sarah-harrington committed:
removed tags from food banks

removed tags from food bank constructors and creation, tags will be associated through junction table dbFBTags now

Jeanneveev committed:
Update editFoodBank.php

Jeanneveev committed:
checks tags if it already exists on the food bank

Jeanneveev committed:
Merge pull request #17 from Jeanneveev/UC-9-V2--View-Tags

personal to group Uc 9 v2  view tags

Jeanneveev committed:
added tag display from database to viewFoodBank.php

Jeanneveev committed:
Merge pull request #18 from Jeanneveev/UC-9-V2--View-Tags

added tag display from database to viewFoodBank.php

jcochran840 committed:
changed changed register.php for admin creation only

jcochran840 committed:
Merge branch 'main' into UC-1-Search1.1

jcochran840 committed:
Merge branch 'main' of https://github.com/lpowell2/FUMC-FB

jcochran840 committed:
added tag image to dashboard, and minor change on personSearch.php

sarah-harrington committed:
select multiple tags functionality for add food bank

lpowell2 committed:
Updated view, edit and search admin. Updated headers to proper terminology.

jcochran840 committed:
Merge branch 'main' into UC-1-Search1.1

lpowell2 committed:
Merge branch 'UC-1-Search1.1' into Update-Admin-Accounts

lpowell2 committed:
Changed volunteer to admin, removed address

sarah-harrington committed:
sql fixes

GraceOrdo committed:
Merge branch 'Multiple-Tags' into dev

Jeanneveev committed:
Merge pull request #1 from lpowell2/UC-9-V2-View_Tags

Uc 9 v2 view tags

GraceOrdo committed:
Merge remote-tracking branch 'origin/Update-Admin-Accounts' into dev

GraceOrdo committed:
Update importFoodBank.php

Jeanneveev committed:
proper mult viewFB

Jeanneveev committed:
multi-edit tags

Jeanneveev committed:
Merge branch 'dev' of https://github.com/lpowell2/FUMC-FB into development

sarah-harrington committed:
delete tag page error fix

sarah-harrington committed:
Merge branch 'dev' of https://github.com/lpowell2/FUMC-FB into dev

Jeanneveev committed:
Merge branch 'dev' into dev

Jeanneveev committed:
Merge pull request #19 from Jeanneveev/dev

Dev

lpowell2 committed:
Simple access and view fixes

lpowell2 committed:
Changed Login Verbiage

lpowell2 committed:
Removed duplicate find person, reformatted

sarah-harrington committed:
fixed delete tag, limited search results

-delete tag works
- search results are limited to entries with position food bank

jcochran840 committed:
Merge branch 'Fix-DeleteTag' into UC-1-Search1.1
